Makefile

```diff
@@ -26,7 +26,7 @@ JSONNET_LOCAL_OR_INSTALLED=$(if $(shell which jsonnet 2>/dev/null),$(shell which
 JSONNETFMT_LOCAL_OR_INSTALLED=$(if $(shell which jsonnetfmt 2>/dev/null),$(shell which jsonnetfmt 2>/dev/null),$(JSONNETFMT_BIN))
 JB_BIN=$(BIN_DIR)/jb
 JSONNET_SRC=$(shell find ./jsonnet -path ./jsonnet/vendor -prune -false -o -type f \( -name "*.jsonnet" -o -name "*.libsonnet" \))
-BENCHMARK_RESULTS=$(shell find ./benchmark -type f -name '*.json')
+BENCHMARK_RESULTS=$(shell find ./telemeter-benchmark -type f -name '*.json')
```

Comment:
I don't think this is right. The telemeter-benchmark binary is not what we are looking for here, but rather we are looking for JSON files in a directory called `benchmark`

Reply:
Got it, my bad, let me revert the second commit